### PR TITLE
Receive message endpoint for onion dev

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@hodgestar, @TrevorOctober ready for review in the morning

File: mentorship/comms/views.py [hunk 1]
```diff
@@ -122,10 +122,41 @@ def post(self, request, format=None):
...
+            )
+
+        recipient = mentor_models.Mentor.objects.get(mentee=sender)
+        mentor_models.Message.objects.create(sender=sender,
+                                             recipient=recipient,
+                                             content=content)
+        mentor_models.Event.objects.create(
+            description="Received message: '" + content + "'",
+            event_type="receive_message",
+            relevant_mentor=recipient.mentor)
```
Comment (hodgestar) [inline mentorship/comms/views.py]: If a mentor can have more than one mentee (not sure if this is allowed) how will they know which mentee this message is from?

**Comment by hodgestar** [comment]
One question, otherwise look good to me.

File: mentorship/comms/views.py (hunk 1 above)
Comment (KaitCrawford) [inline mentorship/comms/views.py]: I'm reasonably sure mentor's can only have one mentee but I'll check. Either way, we store the sender with the message so we can display it however we like.

Comment (KaitCrawford) [inline mentorship/comms/views.py]: Confirmed, the relationship will always be one-to-one

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**